core-bundle/src/Resources/contao/classes/Ajax.php

```diff
@@ -353,24 +353,12 @@ public function executePostActions(DataContainer $dc)
 
 				// Set the new value
 				$varValue = Input::post('value', true);
-
-				switch ($this->strAction)
+				$strKey = match ($this->strAction)
```

Comment:
We cannot use `match()` in Contao 4.13, I‘m afraid.

Reply:
🙈

Reply:
Fixed.